apps/expo/src/components/OutlinedButtons.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Text, TouchableOpacity, View } from "react-native";
+
+interface OutlinedButtonProps {
+  onPress: () => void;
+  children: React.ReactNode;
+  fill: boolean;
+  // icon to be added
+}
+
+const OutlinedButton: React.FC<OutlinedButtonProps> = ({
+  onPress,
+  children,
+  fill,
+}) => {
+  return (
+    <TouchableOpacity onPress={onPress}>
+      <View className="h-12 flex-auto items-start gap-2">
+        <View
+          className={`border-p-40 rounded-full border bg-white px-6 py-2 ${
```

Comment:
remove nested views

Reply:
okay sounds good! should i also change how FilledButtons is structured?

Reply:
Ryan already made changes to those ones, so you can just modify the Outlined one